frontend/styles/globals.css

```diff
@@ -38,3 +38,19 @@ a:visited {
   color: var(--linkBlue);
   text-decoration: underline;
 }
+
+table {
+  border-collapse: collapse;
+}
+
+td,
+thead,
+tfoot {
+  font-family: Inter;
+  height: 36px;
+  text-align: center;
+}
+
+tr:nth-child(odd) {
```

Comment:
If these styles are something specific to the table you're building (as opposed to something general to all tables we could want to build across the app) they should probably be scoped locally to your component - rather than in the global stylesheet

Reply:
I'm going to need more time with this one: I've run into the first major snag with CSS modules, which is that they make using combination selectors in that paradigm effectively impossible.